src/index.html

```diff
@@ -7,12 +7,76 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <a
+        class="card__link-img"
+        href="#product"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="Product image"
+        />
+      </a>
+
+      <a
+        class="card__link-description"
+        href="#product"
+      >
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </a>
```

Comment:
The image and title have the same link, I don't think you need to do it as a link. It's a title, use some heading such as `h2`/`h3`

Reply:
Fixed